isisd/isis_route.c

```diff
@@ -554,6 +554,7 @@ void isis_route_verify_merge(struct isis_area *area,
 						ISIS_ROUTE_FLAG_ZEBRA_SYNCED
 					);
```

Comment:
I think there needs to be a call to `mrnode->info = rnode->info` here, otherwise an active L2 route will not replace an inactive L1 route.